androidsdk/src/main/java/com/wootric/androidsdk/SurveyManager.java

```diff
@@ -96,12 +93,14 @@ public static SurveyManager getSharedInstance() {
     synchronized void stop() {
         handler.removeCallbacks(surveyRunner);
 
-        surveyRunning = false;
+        resetSurvey();
```

Comment:
With the `resetSurvey()` method settings `surveySupportFragment` and `surveyFragment` attributes to null, the fragments will never get dismissed by the following `if / else if`.
If would modify this code to the following:
```
    synchronized void stop() {
        handler.removeCallbacks(surveyRunner);

        if (surveySupportFragment != null) {
            surveySupportFragment.dismiss();
        } else if (surveyFragment != null) {
            surveyFragment.dismiss();
        }
        resetSurvey();
        Log.d(Constants.TAG, "Survey stopped");
    }
```